goalgorilla/drupal_social
# Feature/ds 1545 image editor

How to test:

- reinstall (should get new vars from theme settings)
- Open a node or profile and edit it, so you get a ckeditor
- insert an image

- [x] Theming looks good
- [x] Radio buttons are good
